setuptools/command/upload_docs.py

```diff
@@ -2,7 +2,7 @@
 """upload_docs
 
 Implements a Distutils 'upload_docs' subcommand (upload documentation to
-PyPI's pythonhosted.org).
+sites other than PyPi such as devpi).
```

Comment:
Thanks for taking extra time to clean up docs too!

Reply:
I noticed some other places in `upload_docs` I could change the documentation too. I think it would help decrease the confusion about `upload_docs` being deprecated or not